modules/ITK/cli/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+#the idea here is to have an X.cxx and a X.md where X denotes the cli-command name.
+set( _cli_files
+	statismo-build-shape-model
+	statismo-build-deformation-model
+	statismo-build-gp-model
+	statismo-sample
+)
+
+if(PANDOC)
+	message(STATUS "Found pandoc - generating documentation for the CLI tools")
+	#configuring man page install dir
+	if(WIN32 AND NOT CYGWIN)
+		#put the directory with the manuals in the same directory as the binaries on windows
+		set(CLIDOC_INSTALL_DIR ${INSTALL_BIN_DIR})
+		set(CLIDOC_OUTPUT_DIR "${CMAKE_BINARY_DIR}/doc-cli" )
+	else()
+		#use cmake to find the man pages directory on *NIX
+		include(GNUInstallDirs)
+		if(CMAKE_INSTALL_MANDIR)
+			set(CLIDOC_INSTALL_DIR ${CMAKE_INSTALL_MANDIR})
+		else()
+            set(CLIDOC_INSTALL_DIR ${INSTALL_BIN_DIR})
+			message(STATUS "Couldn't find the man install directory - man pages will NOT be installed properly")
+		endif()
+		set(CLIDOC_OUTPUT_DIR "${CMAKE_BINARY_DIR}/man8" )
+	endif()
+	
+else()
+	message(STATUS "Pandoc not found - documentation will not be generated for the CLI tools")
+endif()
+
+foreach( e ${_cli_files} )
+		add_executable( ${e} "${e}.cxx" )
+		target_link_libraries( ${e} statismo_core ${ITK_LIBRARIES} ${Boost_LIBRARIES})
+		install(TARGETS ${e}
+			DESTINATION ${INSTALL_BIN_DIR}
+		)
+endforeach()
+
+unset(CLIDOC_TARGET_DEPENDENCIES)
```

Comment:
btw these variables need to exist only if PANDOC (and if you want the documentation)